tools/c7n_azure/c7n_azure/query.py

```diff
@@ -153,6 +153,7 @@ def __repr__(cls):
 
 @six.add_metaclass(TypeMeta)
 class TypeInfo(object):
+    groups = ['Other']
```

Comment:
preference for documenting the attribute here and also being explicit that this is doc related in the attribute name, ie. doc_groups